Documentation/kubernetes/install.rst

```diff
@@ -50,8 +50,8 @@ filesystem to be automatically mounted when the node boots.
 If you are using systemd to manage the kubelet, another option is to add a
 mountd systemd service on all hosts:
 
-Due to how systemd mounts <link rel="this SO article for example"
-href="https://unix.stackexchange.com/questions/283442/systemd-mount-fails-where-setting-doesnt-match-unit-name">
+Due to how systemd `mounts
+<https://unix.stackexchange.com/questions/283442/systemd-mount-fails-where-setting-doesnt-match-unit-name>`__
 This string must be reflected in the unit filename.
```

Comment:
We can change this to `.. literalinclude:: ../contrib/systemd/sys-fs-bpf.mount`

Reply:
I thought the reason we stopped doing that earlier was that we didn't want the `cat <<EOF` bit in the file but there is no way to insert the file contents into the block?

Reply:
Ohhh I see, ok then